tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist = lint,py27,py34,py35,pypy,cover
+
+[testenv]
+deps =
+  flask
+  mock
+  pytest
+  pytest-cov
+  pytest-localserver
+  urllib3
+commands =
+  py.test --cov=google.auth --cov=google.oauth2 --cov=tests {posargs:tests}
+
+[testenv:cover]
+basepython = python3.5
+commands =
+  py.test --cov=google.auth --cov=google.oauth2 --cov=tests --cov-report= tests
+  coverage report --show-missing --fail-under=100
```

Comment:
The fail-under? Possibly, but we may have to run separate tests with app engine later.